command/run.go

```diff
@@ -194,6 +203,34 @@ func (r *Run) Execute(args Args, config *config.Couper, logEntry *logrus.Entry)
 	return nil
 }
 
+func readCertificateFile(file string) ([]byte, error) {
+	cert, err := ioutil.ReadFile(file)
+	if err != nil {
+		return nil, fmt.Errorf("error reading ca-certificate: %v", err)
+	} else if len(cert) == 0 {
+		return nil, fmt.Errorf("error reading ca-certificate: empty file: %q", file)
+	}
+
+	pemCerts := cert[:]
+	for len(pemCerts) > 0 {
+		var block *pem.Block
+		block, pemCerts = pem.Decode(pemCerts)
+		if block == nil {
+			return nil, fmt.Errorf("error parsing pem ca-certificate: missing pem block")
+		}
+		if block.Type != "CERTIFICATE" || len(block.Headers) != 0 {
```

Comment:
I have no idea why exactly but since this is the code the certificate bytes have to pass later on while appending them to the transport tls certPool: https://github.com/golang/go/blob/master/src/crypto/x509/cert_pool.go#L206

Call on our site: https://github.com/avenga/couper/pull/447/files#diff-61003b63fe01ebd5063ad7e782e4d4d60ac5d041dae9842ce8f88a5e93f55b0cR54

This has been extracted to the run-section to be able to have better errors instead of true/false with a temporary certPool on startup.